spec/features/create_object_h2_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Use H2 to create an object', type: :feature do
+  let(:collection_title) { RandomWord.nouns.next }
+  let(:item_title) { "SUL Logo for #{collection_title}" }
+  let(:start_url) { "#{Settings.h2_url}/dashboard" }
+  let(:user_email) { "#{AuthenticationHelpers.class_variable_get(:@@username)}@stanford.edu" }
```

Comment:
Have you tried `AuthenticationHelpers.username` rather than `class_variable_get`

Reply:
That works, also updated line 26 to use `AuthenticationHelpers.username`

Reply:
Done